libraries/ms-drivers/test/test_drv120_relay.c

```diff
@@ -1,15 +1,26 @@
 // Brief test sequence for DRV120
 
+#include "delay.h"
 #include "drv120_relay.h"
 #include "log.h"
 #include "ms_test_helpers.h"
 
+// Gpio_it callback function
+static bool s_gpio_it_callback_called = false;
+void prv_callback(const GpioAddress *address, void *context) {
+  LOG_DEBUG("DRV120 Interrupt Triggered!\n");
+  s_gpio_it_callback_called = true;
+}
+
 // Pin used to enable/disable relay
 static const GpioAddress s_test_drv120_pin = { GPIO_PORT_A, 8 };
+static const GpioAddress s_test_drv120_status = { GPIO_PORT_A, 9 };
```

Comment:
If we're in the business of sneakily choosing "test" gpio pins to align with the real ones, I'd suggest making this PA6, since that's the real status pin.